kernelci.org/content/en/docs/contrib.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Contributing Guidelines"
+date: 2022-12-08T09:21:56Z
+draft: false
+weight: 2
+---
+
+KernelCI core project is open for contributions. Contributions may consist of
+adding new build, tests and device types as well as features and bugfixes for
+KernelCI core tools.
+The best way to contribute is to send a PR to [kernelci-core](https://github.com/kernelci/kernelci-core).
+When the PR is created, [KernelCI staging](https://kernelci.org/docs/instances/staging)
+instance takes care of updating the [staging.kernelci.org branch](https://github.com/kernelci/kernelci-core/tree/staging.kernelci.org).
+In general the branch is updated every 8h and a limited set of builds and tests
+is run on it. More detailed information about the logic behind staging runs can
+be found [here](https://kernelci.org/docs/instances/staging).
+
+There are several guidelines which can facilitate the PR review process.
+
+1. Make sure the PR is well described
+   1. Describe the purpose of the changes
+   2. Example use cases are welcome
+2. Attach staging build/test results when possible
+   1. Staging instance is updated every 8h. If the PR is expected to produce build/test results
+   check [staging dashboard](https://staging.kernelci.org) and make sure these are mentioned in the PR comment
```

Comment:
Perhaps an example of how the ideal PR comments and what sort of link to staging is expected would be useful. In particular, the links may help to understand the kind of URLs/paths expected.

Also:

```suggestion
   1. The staging instance is updated every 8h. If the PR is expected to produce build/test results, then
   check [staging dashboard](https://staging.kernelci.org) and make sure these are mentioned in the PR comment
```

Reply:
Actually, I looked for a PR that could be put here as an example, but all I looked into were using links to logs on stored on staging. I decided to not put it here as it was agreed that we're going to either paste log excerpts to the comments or feed data to  pastebin and paste links from there.